plugins/GSdx/GSdx.cpp

```diff
@@ -373,7 +374,7 @@ void GSdxApp::Init()
 	m_default_configuration["linear_present"]                             = "1";
 	m_default_configuration["MaxAnisotropy"]                              = "0";
 	m_default_configuration["mipmap"]                                     = "1";
-	m_default_configuration["mipmap_hw"]                                  = "0";
+	m_default_configuration["mipmap_hw"]                                  = "-1";
```

Comment:
You still need to change the value in GSdx.rc, the dialog window is always initialized with Off as the default value.

Also, I don't think it's necessary (probably not good practice either) to start at `-1` in this case. I would just change it to `0` and adjust the other values accordingly.

Reply:
The code in GSdx.rc controls where the button is placed.

Did you delete your GSdx.ini before doing the testing ?